package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fluidms",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
As this requires changes to the toolchain of both the library and affects users, it is a breaking-change rather than a patch version.

Since the lib was not released as stable yet `v0.*` we might get away with it as it can be considered as a pre-release with an unstable API.

Despite lib version bump (multiple options from minor to major release + breaking changes), it is vital to communicate it transparently with changelog and/or release notes.

Reply:
I appreciate if you could help me (how) to document these changes for all users, cause I am very unfamiliar with this 🥴